perception_dataset/utils/rosbag2.py

```diff
@@ -170,11 +170,13 @@ def pointcloud_msg_to_numpy(pointcloud_msg: PointCloud2) -> NDArray:
 
     # Extract the x, y, z coordinates and additional fields if available
     xyz = points["xyz"]
+    points_arr = xyz
```

Comment:
It looks OK to assign directly.
```suggestion
    points_arr = points["xyz"]
```

Reply:
https://github.com/tier4/tier4_perception_dataset/pull/177/commits/73ee7b277cb1fd5826d01a101846cb7292af0414